docs/userstories/README.md

```diff
@@ -0,0 +1,64 @@
+# Network Policy API User Stories
+
+This directory should be the entry point for any new object or feature proposed to the 
+sig network policy api subgroup 
+
+It is meant to house all user stories that have been reviewed and agreed upon by the sig for any new
+k8's networking security object. 
+
+The model should follow a standard github workflow, with a folder representing proposed new objects which
+contains individual user stories that can be reviewed and commented on. To submit a new user story please 
+open a PR roughly following the [template](#template) shown below. Each story should be in it's own markdown
+file titled with a descriptive 3 word title:  `<Word1>_<Word2>_<Word3>.md, ...`
+
+### Template
+
+```markdown
+
+### Summary 
+
+A user story should typically have a summary structured this way:
+
+1. **As a** [user concerned by the story]
+2. **I want** [goal of the story]
+3. **so that** [reason for the story]
+
+The “so that” part is optional if more details are provided in the description. 
+
+### Description 
+
+The user story should have a reason to exist: what do I need as the user described in the summary?
+This part details any detail that could not be passed by the summary.
+
+### Acceptance Criteria
+
+1. [If I do A.]
+2. [B should happen.]
+
+[
+Also, here are a few points that need to be addressed:
+
+1. Constraint 1;
+2. Constraint 2;
+3. Constraint 3.
+]
+
+### Resources:
+
+* Relavant previous discussions on the story if there are any [i.e link to new story PR]
+* Mockups: [Here goes a URL to or the name of the mockup(s) in inVision];
+
+
+### Notes
+
+[Some complementary notes if necessary:]
+
+* > Here goes a quote from an email
+* Here goes whatever useful information can exist…
+```
+
+### Resources:
+
+* [Style-guides and template for a user story](agile-user-story.md)
```

Comment:
Markdown formatting error ? The link didnt work for me. Should be formatted as \[text\]\(url\)

Reply:
Fixed Thanks!